### PR TITLE
[Release] v0.21.0

### PR DESCRIPTION
**Pull request by jtcohen6**
We released `v0.21.0-rc1` today. If all goes well, we'll be releasing v0.21.0 (final) in 2 weeks (Oct 4).

- [x] Wait for v0.21.0 final release

(Note: I very quickly enabled force pushes in the `next` branch, while I cleaned up commits for an easier review. Full branch protection is back on, including the prohibition of force pushes.)

### CODE REVIEW

File: website/docs/docs/building-a-dbt-project/using-sources.md [hunk 1]
```diff
@@ -164,10 +164,10 @@ Additionally, the `loaded_at_field` is required to calculate freshness for a tab
 These configs are applied hierarchically, so `freshness` and `loaded_at` field values specified for a `source` will flow through to all of the `tables` defined in that source. This is useful when all of the tables in a source have the same `loaded_at_field`, as the config can just be specified once in the top-level source definition.
 
 ### Snapshotting source freshness
```
Comment (jasnonaz) [inline website/docs/docs/building-a-dbt-project/using-sources.md]: Do we want to change the wording around 'snapshotting' here now that we've removed snapshot from the command name?

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md [hunk 2]
```diff
@@ -13,15 +13,15 @@ dbt Cloud provides a helpful interface around dbt's [source data freshness](usin
...
 
-You can add `dbt source snapshot-freshness` anywhere in your list of run steps, but note that if your source data is out of date, this step will "fail', and subsequent steps will not run. dbt Cloud will trigger email notifications (if configured) based on the end state of this step.
+You can add `dbt source freshness` anywhere in your list of run steps, but note that if your source data is out of date, this step will "fail', and subsequent steps will not run. dbt Cloud will trigger email notifications (if configured) based on the end state of this step.
 
-If you *do not* want your models to run if your source data is out of date, then it could be a good idea to run `dbt source snapshot-freshness` as the first step in your job. Otherwise, we recommend adding `dbt source snapshot-freshness` as the last step in the job, or creating a separate job just for this task.
+If you *do not* want your models to run if your source data is out of date, then it could be a good idea to run `dbt source freshness` as the first step in your job. Otherwise, we recommend adding `dbt source freshness` as the last step in the job, or creating a separate job just for this task.
```
Comment (jasnonaz) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md:22]: Would it be helpful to have some information here about how this interaction plays out in the new 'dbt build' command?

File: website/docs/reference/commands/build.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    
+</Changelog>
+
+The `dbt build` command will:
+- run models
+- test tests
+- snapshot snapshots
+- seed seeds
+
+In DAG order, for selected resources or an entire project.
```
Comment (jasnonaz) [inline website/docs/reference/commands/build.md:18]: Should we also clarify the command order? IE which of these takes precedence on a given node in the DAG if multiple apply (looks to be Run,Test,Snapshot which is the order it's in now, just might be helpful to explicitly note).

File: website/docs/reference/commands/rpc.md [hunk 4]
```diff
@@ -210,8 +210,7 @@ All RPC requests accept the following parameters in addition to the parameters l
 
 Several of the following request types accept these additional parameters:
 - `threads`: The number of [threads](configure-your-profile#understanding-threads) to use when compiling (optional)
-- `models`: The space-delimited set of models to compile, run, or test (optional)
-- `select`: The space-delimeted set of resources to seed or snapshot (optional)
+- `select`: The space-delimeted set of resources to execute (optional). (`models` is also supported on some request types for backwards compatibility.)
```
Comment (jasnonaz) [inline website/docs/reference/commands/rpc.md]:
```suggestion
- `select`: The space-delimited set of resources to execute (optional). (`models` is also supported on some request types for backwards compatibility.)
```

File: website/docs/reference/commands/build.md (hunk 3 above)
Comment (jtcohen6) [reply website/docs/reference/commands/build.md:18]: In order to break a DAG "tie," it looks like (in local testing) the implicit order is model, seed, snapshot, test. That's alphabetical order (hah), but I'm not sure where that comes from—if anything, I might expect it to be the order defined [here](https://github.com/dbt-labs/dbt/blob/6854e6746406d87e3e800d7e535c85aac5bf6d2e/core/dbt/task/build.py#L23-L29).

In any event, I think users should imagine the order as non-deterministic between independent nodes that can be parallelized, and very much deterministic for dependencies.

File: website/docs/reference/commands/rpc.md (hunk 4 above)
Comment (jtcohen6) [reply website/docs/reference/commands/rpc.md]: fixed in https://github.com/dbt-labs/docs.getdbt.com/pull/831/commits/fb40346c5952eb505efeca154d5cd6cb2e108a9d

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md (hunk 2 above)
Comment (jtcohen6) [reply website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md:22]: good call! added in https://github.com/dbt-labs/docs.getdbt.com/pull/831/commits/fb40346c5952eb505efeca154d5cd6cb2e108a9d

File: website/docs/docs/building-a-dbt-project/using-sources.md (hunk 1 above)
Comment (jtcohen6) [reply website/docs/docs/building-a-dbt-project/using-sources.md]: yes, well spotted - changed the title in https://github.com/dbt-labs/docs.getdbt.com/pull/831/commits/fb40346c5952eb505efeca154d5cd6cb2e108a9d

**Comment by jtcohen6** [comment]
@jasnonaz Thanks so much for taking a look over! I know there's a lot in this one. If I missed anything big, we've got a generous community of folks who can help us fix it :)

**Pull request merged**